module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -459,8 +459,9 @@ protected function getLocations()
             foreach ($this->getPagedResults(
                 'locations', '/locations'
             ) as $location) {
-                $locationMap[$location->id]
-                    = $location->discoveryDisplayName ?? $location->name;
+                $name = $location->discoveryDisplayName ?? $location->name;
+                $code = $location->code;
+                $locationMap[$location->id] = [$name, $code];
```

Comment:
If you use `compact('name', 'code')` here, you'll get an associative array, which might make the rest of the code more readable (instead of using hard-coded 0/1 indexes).

Reply:
Ah yes. I was on the fence about doing this anyhow. This confirms it for me. Done.